terraform/modules/tf_stream_alert/iam.tf

```diff
@@ -42,6 +42,29 @@ data "aws_iam_policy_document" "rule_processor_invoke_alert_proc" {
   }
 }
 
+// IAM Role Policy: Allow the Rule Processor to publish CloudWatch metrics
+resource "aws_iam_role_policy" "streamalert_rule_processor_cloudwatch_put_metric_data" {
```

Comment:
@austinbyers I think this is a great idea, but for now we'll just get this change in and then think about a larger refactor of shared perms across the 3 functions